gui/templates/rebalancing.html

```diff
@@ -135,9 +135,13 @@ <h5 style="float:left" title="Select outgoing channels above">Manual Rebalance R
     ch.percent_inbound = parseInt(ch.remote_balance*100/ch.capacity)
     ch.percent_outbound = parseInt(ch.local_balance*100/ch.capacity)
     ch.inbound_can = ch.percent_inbound / ch.ar_in_target
-    ch.fee_ratio = ch.local_fee_rate == 0 ? 100 : parseInt(ch.remote_fee_rate*100/ch.local_fee_rate)
+    ch.steps = parseInt(ch.inbound_can < 1 ? 0 :((ch.percent_inbound - ch.ar_in_target)/(ch.ar_amt_target*100/ch.capacity)+0.999))
+    ch.expectedRountingCount = ch.steps + ch.local_balance/ch.ar_amt_target
+
+    ch.cost = ch.remote_fee_rate + ch.steps*ch.remote_base_fee/1000 
+    ch.profit = ch.local_fee_rate + ch.expectedRountingCount*ch.local_base_fee/1000  
```

Comment:
Here profit uses the number of steps it takes to rebalance at current target amount but you wont really know how many txs will yield a base fee in the future (most impact when target amount << average tx amount).

Reply:
The issue I see is that this allows you to potentially lose sats, especially when you lean heavily on a base fee. Unless you are also going to set a max htlc that prevents large txs, you cant be sure you will recover these base fees.